less

```diff
@@ -0,0 +1,44 @@
+note: skipping Xcode version check
+[/Users/drexin/src/swift-source/swift/utils/build-script] NOTE: Using toolchain default
++ mkdir -p /Users/drexin/src/swift-source/build/Ninja-RelWithDebInfoAssert/swift-macosx-x86_64/validation-test-macosx-x86_64/BuildSystem/Output/infer_dumps_deps_if_verbose_build.test.tmp/Ninja-DebugAssert
+-- Build Graph Inference --
+Initial Product List:
+    llvm
+    swift
+    swiftpm
+Final Build Order:
```

Comment:
@drexin I don't think you meant to commit this one.

Reply:
dang it... thanks